augur/tasks/github/detect_move/core.py

```diff
@@ -69,11 +99,7 @@ def ping_github_for_repo_move(augur_db, key_auth, repo, logger,collection_hook='
         'description': f"(Originally hosted at {url}) {old_description}"
     }
 
-    current_repo_dict.update(repo_update_dict)
-
-    result = augur_db.insert_data(current_repo_dict, Repo, ['repo_id'])
-
-    logger.info(f"Updated repo for {owner}/{name}\n")
+    update_repo_with_dict(current_repo_dict, repo_update_dict, logger,augur_db)
```

Comment:
Maybe this is where a repo gets set to be recloned the next time?

Reply:
This updates the repo url with the one provided by github but it doesn't affect facade.